### DISCUSSION TITLE
Can I link my vscode plugin on the homepage?

### DISCUSSION THREAD

**Discussion by Simon-He95** [youzan/vant#12404] [General]
I wrote a vscode plugin, which should bring a better development experience to vant4 users.
I hope more users can experience a great development experience. What do you think? Currently, the vant4 version is supported. If user feedback is good in the future, we will consider supporting version 3. [Link](https://github.com/Simon-He95/vscode-common-intellisense)

**Comment by chenjiahan** [comment]
Of course! You can submit a pull request and add the project to the [Community Ecosystem](https://github.com/youzan/vant#community-ecosystem) part. ❤️

Only support Vant 4 is enough because most Vant 3 users has upgraded to Vant 4.
